docs/source/models.rst

```diff
@@ -2428,6 +2428,7 @@ ZeroShot Classification*          Dict[str, [List[str] | str]*   str or List[str
 Table Question Answering**        Dict[str, [List[str] | str]**  str or List[str]
 Question Answering***             Dict[str, str]***              str or List[str]
 Fill Mask****                     str or List[str]****           str or List[str]
+Feature Extraction                str or List[str]               np.ndarray
```

Comment:
As a follow-up, can we add an example of sentence-transformers to the `examples` repo subdir and then ping the Databricks training team to take a look? They've been asking for this.

Reply:
Definitely. Filing a jira for it now